src/storage/mod.rs

```diff
@@ -892,11 +892,18 @@ impl Storage {
 
                     // no scan_count for this kind of op.
 
+                    let key_len = key.len();
                     snapshot.get(&Key::from_encoded(key))
                         // map storage::engine::Error -> storage::Error
                         .map_err(Error::from)
                         .map(|r| {
-                            thread_ctx.collect_key_reads(CMD, 1);
+                            if let Some(ref value) = r {
+                                let mut stats = Statistics::default();
+                                stats.data.flow_stats.read_keys = 1;
+                                stats.data.flow_stats.read_bytes = key_len + value.len();
+                                thread_ctx.collect_read_flow(ctx.get_region_id(), &stats);
+                                thread_ctx.collect_key_reads(CMD, 1);
```

Comment:
Since you have set read_keys above, do we still need collect_key_reads?

Reply:
collect_key_reads is for `KV_COMMAND_KEYREAD_HISTOGRAM_VEC`. Currently there is a slightly different semantic meaning for those two "keys", mainly in whether failed keys should be counted, so that they are not merged into one interface together.